cli_tools/common/image/importer/api_inflater.go

```diff
@@ -71,21 +73,31 @@ func createAPIInflater(request ImageImportRequest, computeClient daisyCompute.Cl
 }
 
 func (inflater *apiInflater) Inflate() (persistentDisk, shadowTestFields, error) {
+	if inflater.isShadowInflater {
+		return inflater.inflateForShadowTest()
+	}
+
+	ctx := context.Background()
+	startTime := time.Now()
+	diskName := "disk-" + inflater.request.ExecutionID
+
+	cd, err := inflater.createDisk(diskName)
+	if err != nil {
+		return persistentDisk{}, shadowTestFields{}, daisy.Errf("Failed to create disk by api inflater: %v", err)
+	}
+
+	return inflater.getDiskAttributes(ctx, diskName, cd, startTime)
+}
+
+func (inflater *apiInflater) inflateForShadowTest() (persistentDisk, shadowTestFields, error) {
```

Comment:
`inflateAndCalculateChecksum()` ?

Reply:
inflateForShadowTest is a better summary to me, since just as mentiuoned in the above comment, there are some other details different: err msgs, signals. "inflateForShaodeTest" summarixzed the purpose of this function, while "calculateChecksum" is a part of the implementation.